scripts/tutors_script.js

```diff
@@ -8,4 +8,52 @@ $(document).ready(function () {
     $('#tutors-table').DataTable({
         responsive: true
     });
+});
+
+// Event listener for updating checkboxes
+$(".checkbox-big").on("click", function () {
+
+    // Get data for update
+    let column = $(this).data("column");
+    let value = $(this).is(":checked") ? 1 : 0;
```

Comment:
This is cool. If it is checked, the value is 1 and if not the value is 0? (re: ? 1 : 0)

Reply:
Yes